specification/resources/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,59 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+```yaml $(typescript)
+typescript:
+  azure-arm: true
+  batch: true
+  package-name: arm-resource
+  generate-license-txt: true
+  generate-package-json: false
+  generate-readme-md: false
+  output-folder: $(typescript-sdks-folder)/packages/arm-resource
+batch:
+  - package-features: true
+  - package-locks: true
+  - package-policy: true
+  - package-resources: true
+  - package-subscriptions: true
+  - package-links: true
+  - package-managedapplications: true
+```
+
+```yaml $(nodejs) && $(package-features)
```

Comment:
make sure to update all instances of nodejs

Reply:
Thanks.